jdbc/src/java/org/apache/hive/jdbc/HiveBaseResultSet.java

```diff
@@ -542,7 +542,7 @@ private Object evaluate(Type type, Object value) {
       }
       return value;
     case TIMESTAMP_TYPE:
-      return Timestamp.valueOf((String) value);
+      return org.apache.hadoop.hive.common.type.Timestamp.valueOf((String)value);
```

Comment:
Sorry, but why is this Timestamp better? Please fill out the description. (Even for draft :) )

Reply:
not necessarily better, we had an internal discussion about this, I'll update jira but TLDR: we might want to stick to java.sql.Timestamp to remain inline with JDBC spec (e.g. ResultSet.getTimestamp)